package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "argparse": "^1.0.10",
+    "auspice": "~2.0.3",
```

Comment:
@tsibley this is a change from previous behavior which specified the exact auspice version. Could you explain your reasoning here?

Reply:
Ah, yeah, I should have called this change out separately. Since we can be fairly confident in Auspice's adherence to semantic versioning—being it's controlled by us—it seemed useful to allow bug fix releases to happen ~automatically if nextstrain.org is being redeployed anyway.

That said, I'm also completely ok with using a pinned exact version and making any upgrade of Auspice, even a bug fix one, always be an explicit process.

Reply:
Sounds fine to me. Can easily pin if the need arises.

Reply:
This change makes sense to me.